### ISSUE TITLE
The results of the two operations are different when i use standarSqlFormat，the indent before '(' are not the same

### ISSUE THREAD

**Issue by a82448186** [sql-formatter-org/sql-formatter#140]
I have a sql

```
insert into user(name, age) values('text', 11)
```
the first time i format it, i get the result
```
insert into
  user(name, age)
values
('text', 11)
```
Based on the last result，Continue to format，i get the result
```
insert into
  user(name, age)
values
  ('text', 11)
```

i expect the same sql get the same result just like the second
the version i used was 4.0.2

**Comment by nene** [comment]
Problem still present in latest master. Likely related to `values(` looking like a function and getting formatted differently from `values (`.

**Comment by nene** [comment]
This is now finally fixed in 9.0.0-beta1
